### PR TITLE
Created tests and requests for get api/reviews/:review_id/comments

### PR DESCRIPTION
**Pull request by silvester-zogaj**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -116,4 +116,51 @@ describe("/api/reviews", () => {
...
+describe("/api/reviews/:review_id/comments", () => {
+  test("Get - status: 200 - return an array of comments of the given review id", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+
+        comments.forEach((comment) => {
+          expect(comment.hasOwnProperty("comment_id")).toBe(true);
```
Comment (JVal-NC) [inline __tests__/app.test.js:139]: Can we check the data type of each property. Retrospectively the same comment can be applied to the test on line 88.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -116,4 +116,51 @@ describe("/api/reviews", () => {
...
+
+describe("/api/reviews/:review_id/comments", () => {
+  test("Get - status: 200 - return an array of comments of the given review id", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+
+        comments.forEach((comment) => {
```
Comment (JVal-NC) [inline __tests__/app.test.js:138]: Can we check the length or at least for it to be greater than 0 for comments before we use a forEach. If the length is zero the callback function and its assertions wont be run.

If this is the case for previous ticket tests then these should also be updated.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -116,4 +116,51 @@ describe("/api/reviews", () => {
...
+  test("GET - status: 200 - responds with an empty array if review id exists but no comments are found", () => {
+    return request(app)
+      .get("/api/reviews/5/comments")
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+        expect(comments).toEqual([]);
+      });
+  });
+  test("Get - status: 404 - responds with review not found", () => {
```
Comment (JVal-NC) [inline __tests__/app.test.js:173]: Good tests, 1 more that should be included is a bad id, ie a string.

**Review by JVal-NC — COMMENTED** [review]
Some minor things to fix in this PR, but good work so far. Another PR when your ready please.

File: __tests__/app.test.js [hunk 4]
```diff
@@ -116,4 +125,67 @@ describe("/api/reviews", () => {
...
+      .then((response) => {
+        const { comments } = response.body;
+        expect(comments.length).toBe(3);
+        comments.forEach((comment) => {
+          expect(comment.hasOwnProperty("comment_id")).toBe(true);
+          expect(comment.hasOwnProperty("votes")).toBe(true);
+          expect(comment.hasOwnProperty("created_at")).toBe(true);
+          expect(comment.hasOwnProperty("author")).toBe(true);
+          expect(comment.hasOwnProperty("body")).toBe(true);
+          expect(comment.hasOwnProperty("review_id")).toBe(true);
```
Comment (pippaamy) [inline __tests__/app.test.js]: we can expect the review_id here to be 2

**Review by pippaamy — APPROVED** [review]
great job, lets add the expected if to the test :)

**Pull request merged**